package.json

```diff
@@ -6,10 +6,12 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "bootstrap": "^4.5.0",
```

Comment:
you should not be usıng both bootstrap and react-bootstrap. we should only be using react-bootstrap

`yarn remove bootstrap`